src/main.rs

```diff
@@ -10,33 +9,17 @@ use tokio::sync::mpsc;
 struct Cli {
     pattern: String,
 
-    #[clap(parse(from_os_str), multiple_values = true)]
-    paths: Vec<std::path::PathBuf>,
-}
-
-fn convert_paths_to_utf8(paths: Vec<std::path::PathBuf>) -> Vec<Utf8PathBuf> {
-    let mut results: Vec<Utf8PathBuf> = Vec::new();
-    let mut errors = false;
-    for path in paths {
-        match Utf8PathBuf::from_path_buf(path) {
-            Ok(value) => results.push(value),
-            _ => errors = true,
-        }
-    }
-    if errors {
-        warn!(
-            "Discarded some file paths because they had ivalid encoding. Valid file paths: {}",
-            results.iter().map(|p| p.to_string()).collect::<Vec<String>>().join(", ")
-        );
-    }
-    results
+    #[clap(multiple_values = true)]
+    paths: Vec<PathBuf>,
 }
 
 #[tokio::main]
 async fn main() -> Result<()> {
+    tracing_subscriber::fmt::init();
```

Comment:
this is the `tracing` subscriber compatible with `log`, and should be the first thing to be initialized in `main`